x-pack/plugins/observability_shared/public/components/page_template/page_template.tsx

```diff
@@ -194,38 +198,45 @@ export function ObservabilityPageTemplate({
       >
         {({ isTourVisible }) => {
           return (
-            <KibanaPageTemplate
-              restrictWidth={false}
-              {...pageTemplateProps}
-              solutionNav={
-                showSolutionNav
-                  ? {
-                      icon: 'logoObservability',
-                      items: sideNavItems,
-                      name: sideNavTitle,
-                      // Only false if tour is active
-                      canBeCollapsed: isTourVisible === false,
-                    }
-                  : undefined
-              }
-            >
-              <KibanaErrorBoundaryProvider analytics={services.analytics}>
-                <KibanaErrorBoundary>
-                  <KibanaPageTemplate.Section
-                    component="div"
-                    alignment={pageTemplateProps.isEmptyState ? 'center' : 'top'}
-                    {...pageSectionProps}
-                  >
-                    {children}
-                  </KibanaPageTemplate.Section>
-                </KibanaErrorBoundary>
-              </KibanaErrorBoundaryProvider>
-              {bottomBar && (
-                <KibanaPageTemplate.BottomBar {...bottomBarProps}>
-                  {bottomBar}
-                </KibanaPageTemplate.BottomBar>
-              )}
-            </KibanaPageTemplate>
+            <div ref={containerRef}>
```

Comment:
**issue**: hey @shahzad31 sorry to reopen this PR review, apparently this change broke the layout of some pages using it, as this wrapping div block limits the maximum height on the page. In the LogsExplorer and DatasetQuality page, for example:

<img width="3007" alt="Screenshot 2024-02-02 at 12 13 13" src="https://github.com/elastic/kibana/assets/34506779/6dfdc406-3cf9-4770-a248-3db46c02ea0d">

Let me know if you could reproduce it, I can't say how many pages have been impacted by this change, I just found out about the two mentioned pages.